irods/test/query2_test.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+
+import irods.test.helpers as helpers
+
+
+class TestQuery2(unittest.TestCase):
+
+    def setUp(self):
+        self.sess = helpers.make_session()
+
+        if self.sess.server_version < (4, 3, 2):
+            self.skipTest(
+                'GenQuery2 is not available by default in iRODS before v4.3.2.')
+
+        self.coll_path_a = '/{}/home/{}/test_query2_coll_a'.format(
+            self.sess.zone, self.sess.username)
+        self.coll_path_b = '/{}/home/{}/test_query2_coll_b'.format(
+            self.sess.zone, self.sess.username)
+        self.sess.collections.create(self.coll_path_a)
+        self.sess.collections.create(self.coll_path_b)
```

Comment:
Are these collections cleaned up automatically due to the type of thing created by `helpers.make_session()`, or do we need to remove the collections in a `tearDown`?

Reply:
Right, this wasn't cleaned up, Have added a `tearDown` function now.